Purchases/Public/RCPurchases.m

```diff
@@ -89,40 +89,40 @@ + (BOOL)canMakePayments
 
 + (instancetype)configureWithAPIKey:(NSString *)APIKey
 {
-    RCPurchases *purchases = [[RCPurchases alloc] initWithAPIKey:APIKey];
-    [RCPurchases setDefaultInstance:purchases];
-    return purchases;
-}
-
-- (instancetype)initWithAPIKey:(NSString *)APIKey
-{
-    return [self initWithAPIKey:APIKey appUserID:nil];
+    return [RCPurchases configureWithAPIKey:APIKey appUserID:nil];
```

Comment:
i think you want to do `self` here instead of referencing `RCPurchases` directly. Only affects subclassing but its best practice to do.